moodle/Tests/Sniffs/Commenting/fixtures/MissingDocblock/docblock_with_multiline_attributes.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * This file contains multiple testcases for multi line attributes.
+ */
+
+namespace MoodleHQ\MoodleCS\moodle\Tests\Sniffs\PHPUnit;
+
+defined('MOODLE_INTERNAL') || die(); // Make this always the 1st line in all CS fixtures.
+
+/**
+ * Example class. 
+ */
+#[\Attribute(
+    attr1: 'asdf',
+    attr2: 'asdf',
+)]
+class class_multiline_attribute {
+
+    /**
+     * Method attribute.
+     */
+    #[\Attribute(
+        attr1: 'asdf',
+        attr2: 'asdf',
+    )]
+    function method_multiline_attribute(): void {
+    }
+}
+
+/**
+ * Interface with multiline attributes.
+ */
+#[\Attribute(
+    attr1: 'asdf',
+    attr2: 'asdf',
+)]
+interface interface_multiline_attribute {
+}
+
+/**
+ * Trait with multiline attributes.
+ */
+#[\Attribute(
+    attr1: 'asdf',
+    attr2: 'asdf',
+)]
+trait trait_multiline_attribute {
+}
+
+/**
```

Comment:
Do we have any example having a blank line between the end of attribute and artifact declaration? I imagine that it will lead to "not-documented" and/or "attribute-without-target", but from this sniff perspective, maybe it also makes sense to have it covered.

Example:

```
/**
 * Trait with multiline attributes.
 */
#[\Attribute(
    attr1: 'asdf',
    attr2: 'asdf',
)]

trait trait_multiline_attribute {
}
```

Reply:
Ok, was looking to existing ' moodle.Commenting.MissingDocblock` tests, and, it seems that, for classes, we have that case already covered by the `class_only_with_attributes_incorrect_whitespace` fixture.

What I'm going to do is:
1. Add similar cases for method, interface and trait.
2. In the `` fixture, change all the ocurrences of `#[\Attribute(` to `#[someattribute(` because the former is incorrect.

Will add commit and, if everything looks ok, proceed to merge this.

Reply:
Done, I've added a 4th commit with the points commented above.